src/commands/auth/logout.js

```diff
@@ -7,8 +7,12 @@ function * run (context, heroku) {
 }
 
 const cmd = {
-  description: 'clears local login credentials',
-  run: cli.command(co.wrap(run))
+  description: 'clear credentials for the current logged in user',
+  run: cli.command(co.wrap(run)),
+  help: `Example:
+
+    $ heroku auth:logout
+    Local credentials cleared`
```

Comment:
This output should be different since it's actually logging the user out, but this is an issue with the command not your documentation

Reply:
I see what you mean. That was the actual output I captured.